### ISSUE TITLE
[Bug] Abilities that change move type seem to change the Type icons in the UI

### ISSUE THREAD

**Issue by star-krieg** [pagefaultgames/pokerogue#3996]
### Describe the bug

Abilities that change move type (Normalize, Refrigerate, etc.) also change the move type icons in the UI. I don't think this is intentional because this happened before in a different bug fix PR (https://github.com/pagefaultgames/pokerogue/pull/1440#issuecomment-2145857208) but it was fixed before it was merged.

### Session export file

Session data with a Normalize fusion Smeargle
[sessionData4_starkrieg.txt](https://github.com/user-attachments/files/16844742/sessionData4_starkrieg.txt)

### Expected behavior

Expected to see the original move type icons for the moves instead of the type-changed icons.

### Screenshots / Videos

Here are some screenshots:

Amaura with Refrigerate
![Refrigerate_IceIcons](https://github.com/user-attachments/assets/495636e2-91b6-4a64-a0fe-8fa840c15fa5)
Smeargle fused to a Normalize Delcatty
![NormalSmeargle](https://github.com/user-attachments/assets/95002ddd-70c9-4d49-85b5-74da1548d3fc)

### What OS did you observe the bug on?

Mac/OSX

### Which browser do you use?

Chrome

**Comment by innerthunder** [comment]
I intentionally changed this in [this PR](https://github.com/pagefaultgames/pokerogue/pull/3704), which borrowed from [another PR](#3417 ) that made the same changes.

I would say this isn't a bug, but I suppose this could use more input from other devs.

**Comment by SirzBenjie** [comment]
This isn't a bug; it's a feature. This is in parallel with smogon.
